console/src/app/modules/settings-list/settings.ts

```diff
@@ -1,6 +1,15 @@
 import { PolicyComponentServiceType } from '../policies/policy-component-types.enum';
 import { SidenavSetting } from '../sidenav/sidenav.component';
 
+export const ORGANIZATIONS: SidenavSetting = {
+  id: 'organizations',
+  i18nKey: 'SETTINGS.LIST.ORGS',
+  groupI18nKey: 'SETTINGS.GROUPS.GENERAL',
+  requiredRoles: {
+    [PolicyComponentServiceType.ADMIN]: ['org.read'],
```

Comment:
since were using the ListMyProjectOrgs endpoint it would be any authenticated user, but after #7283 it' would require `iam.read`

Reply:
I'd merge #7283 as it is which uses the adminservice to list orgs so that we can change this to `iam.read`,
the auth user can select his orgs from the dropdown on the top left. The button show all orgs will then also appear for 'iam.read` users only.

Reply:
this is set to `iam.read` now